plugins/modules/nd_pcv_delta_analysis.py

```diff
@@ -29,15 +31,15 @@
     aliases: [ fab_name, ig_name ]
   name:
     description:
-    - The name of the pre-change validation.
+    - The name of the Pre-change Analysis.
     type: str
     required: true
-  site_name:
+  fabric:
     description:
-    - Name of the Assurance Entity.
+    - Name of the Fabric.
     type: str
     required: true
-    aliases: [ site ]
+    aliases: [ site, site_name, fabric_name ]
   epoch_choice:
```

Comment:
Shall we update this to snapshot_choice to be consistent with the recent changes made to the other modules?